web/themes/custom/jcc_components/includes/views.inc

```diff
@@ -390,6 +390,15 @@ function jcc_components_preprocess_views_view__documents__default(&$variables) {
 
     // Return published oral arguments if available.
     $latest_oa = array_slice($oral_args, 0, 1);
+    if($latest_oa) {
+      $variables['show_documents'] = FALSE;
+      $variables['card_title'] = $latest_oa[0]->label();
```

Comment:
Strange that you needed to specify the index [0]. But if needed from the array_slice(), okay. I thought the slice would have pulled the first item already but I trust that you might have still needed the index depending on the result returned.

Reply:
Yes index[0] was needed here.